.github/workflows/build.yml

```
@@ -33,6 +33,8 @@ jobs:
         include:
           - {name: Linux, python: '3.9', os: ubuntu-latest}
     env:
+      CIBW_ARCHS_LINUX: "auto aarch64"
+      CIBW_ARCHS_MACOS: "x86_64 universal2 arm64"
```

Notes from https://cibuildwheel.readthedocs.io/en/stable/faq/#universal2:

>The dual-architecture universal2 has a few benefits, but a key benefit to a universal wheel is that a user can bundle these wheels into an application and ship a single binary.
>
>However, if you have a large library, then you might prefer to ship the two single-arch wheels instead - x86_64 and arm64. In rare cases, you might want to build all three, but in that case, pip will not download the universal wheels, because it prefers the most specific wheel available.

---

Suggestion, remove universal2.

```suggestion
      CIBW_ARCHS_LINUX: "auto aarch64"
      CIBW_ARCHS_MACOS: "x86_64 arm64"
```

---

That came from the docs example for [CIBW_ARCHS](https://cibuildwheel.readthedocs.io/en/stable/options/#archs) and is the default for `auto`.